xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/ExportAction.java

```diff
@@ -179,7 +202,7 @@ private String exportHTML(XWikiContext context) throws XWikiException, IOExcepti
             }
 
             DocumentReferenceResolver<String> resolver =
-                Utils.getComponent(DocumentReferenceResolver.TYPE_STRING, "current");
+                    Utils.getComponent(DocumentReferenceResolver.TYPE_STRING, "current");
```

Comment:
I assume this is the result of the automatic formatter using the XWiki code style. It strange that it uses 8 spaces to indent.